### ISSUE TITLE
Address notification banner accessibility concerns

### ISSUE THREAD

**Issue by vanitabarrett** [alphagov/govuk-frontend#2011]
## What
Follow up on the [feedback](https://github.com/alphagov/govuk-frontend/issues/1999#issuecomment-723020026) we got from the accessibility clinic and our own accessibility testing.

## Done when
We've taken some action on all of the concerns below or documented why we don't think changes are needed:

- [x] Users may miss a notification banner at the top of a page if they skip straight to the H1
- [x] Positioning of banner and length of content may mean more scrolling for users on smaller devices/screen magnifiers to reach the main page content
- [x] Autofocus behaviour: remove tabindex on blur so the banner is not focusable on click
- [x] Should an error summary and notification banner ever be used together? An action link in the error summary means it's more likely a user will skip past the banner
- [x] Styling a paragraph as a heading may cause confusion for users who override styles or disable CSS

**Comment by hannalaakso** [comment]
One possible course of action is to add some of these under the
> We need more research to understand ...

section in the [guidance](https://docs.google.com/document/d/1SNRBfwvZGS0Iqa_IfMMKwBXSC4bR_D3XrWCQPcbM9gI/edit#heading=h.kvth2ctbmufd).

**Comment by hannalaakso** [comment]
### Update:
- Users may miss a notification banner at the top of a page if they skip straight to the H1. Positioning of banner and length of content may mean more scrolling for users on smaller devices/screen magnifiers to reach the main page content
 **Action**: We'll be including an example of a banner with long content in our upcoming accessibility research prototype. We'll document any relevant findings in the research section of the guidance.
- Autofocus behaviour: remove tabindex on blur so the banner is not focusable on click
  **Action:** Addressed by https://github.com/alphagov/govuk-frontend/pull/2014.
- Should an error summary and notification banner ever be used together? An action link in the error summary means it's more likely a user will skip past the banner
**Action**: We're going to specify in the guidance that teams shouldn't place an alert notification banner and an error summary on the same page.
- Styling a paragraph as a heading may cause confusion for users who override styles or disable CSS
**Action:** We'll be including an example of a banner with paragraph styled as a heading in our upcoming accessibility research prototype. We'll document any relevant findings in the research section of the guidance.
